_shared/project/requirements/prod.in

```diff
@@ -7,6 +7,10 @@ newrelic
 sqlalchemy
 psycopg2
 alembic
+{% if cookiecutter.get("_directory") == "pyramid-app" %}
+pyramid-tm
+zope.sqlalchemy
+{% endif %}
```

Comment:
These're the dependencies needed to integrate a SQLAlchemy session into the Pyramid request lifecycle.